compiler/AST/ImportStmt.cpp

```diff
@@ -394,6 +376,66 @@ void ImportStmt::validateUnqualified() {
   }
 }
 
+/************************************* | **************************************
+*                                                                             *
+* Determine if the provided type was named in some form in the import         *
+* statement.  Used for determining if we should traverse the import statement *
+* to find its methods.                                                        *
+*                                                                             *
+************************************** | *************************************/
+std::set<const char*> ImportStmt::typeWasNamed(Type* t) const {
+  std::set<const char*> namedTypes;
+
+  typeWasNamed(t, &namedTypes);
+  return namedTypes;
+}
+
+void ImportStmt::typeWasNamed(Type* t,
+                              std::set<const char*>* namedTypes) const {
+  // We don't define any symbols for unqualified access, so the type was not
+  // listed
+  if (!providesUnqualifiedAccess()) {
+    return;
+  } else {
+    std::string nameToCheck = t->symbol->name;
```

Comment:
Why bother with `std::string` here? It is always an astr so leave it `const char*` ?